php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1110.html

```diff
@@ -0,0 +1,23 @@
+<p>The use of parentheses, even those not required to enforce a desired order of operations, can clarify the intent behind a piece of code. But
+redundant pairs of parentheses could be misleading, and should be removed. </p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+$x = ($y / 2 + 1); // Compliant even if the parenthesis are ignored by the compiler
```

Comment:
@m-g-sonar I updated the RSPEC to have valid PHP code and I updated this PR accordingly.

Reply:
@pynicolas Oups sorry, I forgot to check about that, indeed! Thanks a lot!